### DISCUSSION TITLE
GitHub Student pack

### DISCUSSION THREAD

**Discussion by Nightfury874** [community/community#22088] [New to GitHub]
<p>I have applied for GitHub student pack around 2 weeks back and haven’t heard from them yet (they did reply to my email telling they were short on men and there was no average wait time for it).<br>
I wanted to know if I should cancel my old request and submit a new request.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome</p>
<p>Opening a new request would substantially delay your reply time so I would strongly recommend against it unless you feel you gave inaccurate information or didn’t follow the instructions in the first one.</p>
